extensions/applicationinsights-react-js/package.json

```diff
@@ -47,8 +47,8 @@
     },
     "dependencies": {
         "@microsoft/applicationinsights-shims" : "1.0.0",
-        "@microsoft/applicationinsights-core-js": "2.5.4",
-        "@microsoft/applicationinsights-common": "2.5.4",
+        "@microsoft/applicationinsights-core-js": "2.5.5",
```

Comment:
Since this plugin is now decoupled from rush, need to use semver carets for now.